scripts/update_chrome_shortcut.py

```
@@ -21,42 +27,72 @@ def _update_shortcut(shortcut: Path, shell) -> bool:
-    return updated
+    """Update Chrome shortcuts and return modified paths."""
+    if os.name == "nt" and win32com is not None:
+        shell = win32com.client.Dispatch("WScript.Shell")
+        locations = [
+            Path(os.environ.get("USERPROFILE", "")) / "Desktop",
+            Path(os.environ.get("APPDATA", ""))
+            / "Microsoft"
+            / "Windows"
+            / "Start Menu"
+            / "Programs",
+            Path(os.environ.get("ProgramData", ""))
```

Ruff (0.11.9)

Use capitalized environment variable `PROGRAMDATA` instead of `ProgramData`

(SIM112)